_posts/2000-01-05-install.md

```diff
@@ -128,11 +128,11 @@ Examples:
 
 - **Aura**:
   ```
-  sudo aura -A vscodium
+  sudo aura -A vscodium-bin
   ```
 - **Yay**:
   ```
-  sudo yay -S vscodium
+  sudo yay -S vscodium-bin
```

Comment:
I don't know about aura but yay *should not* be run with sudo. The instruction should be:

`yay -S vscodium-bin`